standard/classes.md

```diff
@@ -4188,12 +4188,12 @@ An *indexer_declaration* may include a set of *attributes* ([§22](attributes.md
 
 Indexer declarations are subject to the same rules as method declarations ([§15.6](classes.md#156-methods)) with regard to valid combinations of modifiers, with the one exception being that the `static` modifier is not permitted on an indexer declaration.
 
+The modifiers `virtual`, `override`, and `abstract` are mutually exclusive except in one case. The `abstract` and `override` modifiers may be used together so that an abstract indexer can override a virtual one.
```

Comment:
This was omitted here, and similarly in §15.7.1 for properties, as it is in §15.6.1 referenced in the preceding para (as §15.6 both here and in properties) – bullets 3 & 5 of the list describing the valid combinations.

What isn’t remarked on in §15.6.1 is that `abstract` and `override` used together allow an abstract *member* to override a virtual one. If that remark needs to be made it should be made in §15.6.1 and not just here.

So I suggest removing this added para here and, if felt remark-worthy, adding to ¢15.6.1 that abstract members can override virtual ones – it falls out of the bulleted list but may not be obvious.